_data-prepper/pipelines/configuration/processors/otel-trace-group-processor.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: OpenTelemetry trace group processor
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# OpenTelemetry (OTel) trace group processor
+
+The `OTel trace group` processor completes missing trace group related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/834f28fdf1df6d42a6666e91e6407474b88e7ec6/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend. The `OTel trace group` processor iedntifies the missing trace group information for a `spanId` by looking up the relevant fields in its root `span` stored in OpenSearch, or in the Amazon OpenSearch Service backend that the local Data Prepper host ingests.
+
+## Basic usage
+
+<!---Other sections introduce a YAML file here. Is this section missing that example?--->
+
+### OpenSearch
+
+<!---Need an introduction here--->
```

Comment:
If there isn't a sentence here, why do we have this in doc review? this needs. to be here